prbt_hardware_support/include/prbt_hardware_support/modbus_msg_brake_test_wrapper.h

```diff
@@ -73,9 +76,19 @@ inline bool ModbusMsgBrakeTestWrapper::hasBrakeTestRequiredFlag() const
   return hasRegister(getApiSpec().getRegisterDefinition(modbus_api_spec::BRAKETEST_REQUEST));
 }
 
-inline bool ModbusMsgBrakeTestWrapper::isBrakeTestRequired() const
+inline IsBrakeTestRequiredResponse::_result_type ModbusMsgBrakeTestWrapper::getBrakeTestRequirementStatus() const
 {
-  return getRegister(getApiSpec().getRegisterDefinition(modbus_api_spec::BRAKETEST_REQUEST));
+  switch(getRegister(getApiSpec().getRegisterDefinition(modbus_api_spec::BRAKETEST_REQUEST)))
+  {
+    case REGISTER_VALUE_BRAKETEST_NOT_REQUIRED:
+      return IsBrakeTestRequiredResponse::NOT_REQUIRED;
+
+    case REGISTER_VALUE_BRAKETEST_REQUIRED:
+      return IsBrakeTestRequiredResponse::REQUIRED;
+
+    default:
+      return IsBrakeTestRequiredResponse::UNKNOWN;
```

Comment:
when will this trigger? `getRegister(getApiSpec().getRegisterDefinition(modbus_api_spec::BRAKETEST_REQUEST))` can either return 0 or 1, right?

Reply:
https://github.com/PilzDE/pilz_robots/blob/4bd0e90391b5513abbb7a6430324f6c430de4185/prbt_hardware_support/include/prbt_hardware_support/modbus_msg_wrapper.h#L80

Reply:
ok